content_templates/templates/common/instance_view_builder.html

```diff
@@ -16,9 +16,79 @@
         {{ build_instance_editorial_part(graph, instance_uri) }}
         {{ build_lam_property_configuration_list(graph, instance_uri, level + 1) }}
         {{ build_lam_classification_hint_list(graph, instance_uri, level + 1) }}
+        {{ build_useful_links(instance_uri) }}
     </section>
 {% endmacro %}
 
+
+{% macro build_useful_links(instance_uri) %}
+    <p class="debug">build_useful_links <strong>{{ instance_uri }}</strong></p>
+
+    {% set graph = conf.graphs.lam_classes %}
+
+    {{ build_used_in_classify_with_or_has_property_configuration(graph, instance_uri) }}
+    {{ build_constraints(graph, instance_uri) }}
+
+{% endmacro %}
+
+
+{% macro build_used_in_classify_with_or_has_property_configuration(graph, instance_uri) %}
+    <p class="debug">build_used_in_classify_with_or_has_property_configuration <strong>{{ instance_uri }}</strong></p>
+    {% set select_query =
+    "PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+    prefix lam: <http://publications.europa.eu/ontology/lam-skos-ap#>
+
+    select distinct ?link ?linkLabel
+    {
+      values ?instanceUri {<"~instance_uri~">}
+      graph <"~graph~">
+      {
+        ?propConfig lam:path ?instanceUri .
+        optional {
+          ?link lam:classifyWith | lam:hasPropertyConfiguration ?propConfig.
```

Comment:
I think that this 2 sections should be separated. I would check with @costezki

Reply:
Where is this macro used? 
If it is used in rendering a LAMClass then they should be separated sections.
If it is used in rendering a section of Quick-links then they can be merged.

Reply:
it's used in generating the quick links